### ISSUE TITLE
install BBB with certum ssl

### ISSUE THREAD

**Issue by mhsprog** [bigbluebutton/bbb-install#538]
Hi
Is it possible to install the BigBlueButton and green light with the Certum SSL?

**Comment by ffdixon** [comment]
If you use the `-d` option, you can provide your own certificate.  See

  https://github.com/bigbluebutton/bbb-install#command-options

See also

  https://github.com/bigbluebutton/bbb-install/blob/master/bbb-install-2.5.sh#L787

**Comment by mhsprog** [comment]
> If you use the `-d` option, you can provide your own certificate. See
> 
> https://github.com/bigbluebutton/bbb-install#command-options
> 
> See also
> 
> https://github.com/bigbluebutton/bbb-install/blob/master/bbb-install-2.5.sh#L787

Thanks, I'll try it
And the next question is that I want to install it on the private network for the university. Can I use self signed certificate?

**Comment by lonesomewalker** [comment]
Yes you can, but then you have to add these to the server as trusted.
